MBC_RegistrationMobile.class.inc

```diff
@@ -63,12 +63,8 @@ class MBC_RegistrationMobile
     $payloadDetails = unserialize($payload->body);
     $deliveryTag = $payload->delivery_info['delivery_tag'];
 
-    // @todo - remove $payloadDetails['activity'] check when this consumer handles all Mobile Commons signups
-    if ((isset($payloadDetails['mobile']) && $payloadDetails['mobile'] != '') &&
-        ($payloadDetails['activity'] == 'user_welcome-niche' ||
-         $payloadDetails['activity'] == 'user_import' ||
-         $payloadDetails['activity'] == 'cgg2014_signup' ||
-         $payloadDetails['activity'] == 'cgg2014_vote')) {
+    if (!empty($payloadDetails['mobile']) &&
```

Comment:
Still confused when to use `isset()` vs `empty()` but it looks good. +1

Ref: https://www.virendrachandak.com/techtalk/php-isset-vs-empty-vs-is_null/